### ISSUE TITLE
Always have to resolve merge conflicts when merging into main

### ISSUE THREAD

**Issue by chadstewart** [TechIsHiring/techishiring-website#16]
Whenever I try to create a PR into main, I always have to resolve merge conflicts which is just accepting current.

Need to find out why that is so I can have more frictionless releases.

**Comment by stradox4u** [comment]
- Looking at your git history, it seems like your merging process might be to blame, as you'll see that your main branch is far from, and has a different history from your dev branch.
- At my job, we merge our features into dev, and then dev into main. We make sure that main and dev share a common commit history, so there are no merge conflicts at this step.

**Comment by stradox4u** [comment]
Yeah, I think you should only squash when merging a feature into dev.

From dev to main, there should be no squashing, so their commit histories
stay identical.
